src/python/bot/tasks/task_creation.py

```diff
@@ -142,13 +142,23 @@ def create_regression_task_if_needed(testcase):
   tasks.add_task('regression', testcase.key.id(), testcase.job_type)
 
 
-def create_stack_task_if_needed(testcase):
-  """Creates a stack task if needed."""
-  second_stack_job_type = environment.get_value('SECOND_STACK_JOB_TYPE')
-  if not second_stack_job_type:
-    return
+def create_variant_tasks_if_needed(testcase):
+  """Creates a variant task if needed."""
+  testcase_id = testcase.key.id()
+  jobs = ndb_utils.get_all_from_model(data_types.Job)
```

Comment:
nit: I'm thinking if we can do some extra filtering at this step, like `name != testcase.job_type`, but that feels insufficient. If we could check the project, that might be a good optimization I think (as OSS-Fuzz has a ton of jobs / projects), but we don't have project name in the Job entity as is.

Reply:
every oss-fuzz job has project name, e.g. "PROJECT_NAME = bignum-fuzzer"
also, i filter by project name inside the loop.

Reply:
Sorry, my point was that with current approach we're querying all the jobs from the datastore and then we're skipping like 99.5% of them. If we could avoid doing that and do the filtering inside the datastore query, it might be more efficient. I don't know whether it's a big gain or not, that's why it's just a nit :)

Reply:
It cannot be done in query, since the things we are looking for are in the job environment string. like PROJECT_NAME, EXPERIMENTAL, etc.